src/main/resources/archetype-resources/groovy-pom.xml

```diff
@@ -44,20 +45,35 @@
         <maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
         <groovy-maven-plugin.version>2.1.1</groovy-maven-plugin.version>
         <groovy-eclipse-compiler.version>3.7.0</groovy-eclipse-compiler.version>
-        <groovy-eclipse-batch.version>#if( $bonitaVersion.matches('7.(1[3-9]|[2-9][0-9]).*') || $bonitaVersion.matches('[8-99].[0-9]+.*') )
+        <groovy-eclipse-batch.version>#if( $after713 )
 3.0.7-03#else
```

Comment:
3.0.8-01

Reply:
changed groovy-eclipse-batch version to 3.0.8-01